addons/binding/org.openhab.binding.vitotronic/ESH-INF/thing/thing-types.xml

```diff
@@ -311,147 +526,61 @@
-    </channel-type>
-         
+	
     <!-- Channel Types for circuit -->   
     
-    <channel-type id="flow_temp"> 
-        <item-type>Number</item-type>
-        <label>Flow</label>
-        <description>Temperature sensor of the ciruit flow</description>
-        <category>Temperature</category>
-        <state pattern="%.1f °C" readOnly="true"/>
-    </channel-type>
-    
-    <channel-type id="pump">
-        <item-type>Switch</item-type>
-        <label>Pump</label>
-        <description>Pump state</description>
-        <state readOnly="true"/>
-    </channel-type>
-
-    <channel-type id="operationmode" advanced="true">
+    <channel-type id="operationmode">
         <item-type>Number</item-type>
         <label>Operationmode</label>
         <description>Operationmode</description>
-        <state pattern="%d" readOnly="true"/>
-    </channel-type>
-    
-    <channel-type id="partymode">
-        <item-type>Switch</item-type>
-        <label>Partymode</label>
-        <description>Partymode on/off</description>
-        <state readOnly="false"/>
+        <state pattern="%d" readOnly="false" max="4" min="0" step="1"/>
     </channel-type>
     
-    <channel-type id="savemode">
-        <item-type>Switch</item-type>
-        <label>Savemode</label>
-        <description>Savemode on/off</description>
-        <state readOnly="false"/>
-    </channel-type>
-    
-    <channel-type id="party_temp" advanced="true">
+    <channel-type id="party_temp_setpoint" advanced="true">
         <item-type>Number</item-type>
-        <label>Party</label>
+        <label>Party Mode Temperature Setpoint</label>
         <description>Target temperature of party mode</description>
         <category>Temperature</category>
-        <state min="4.5" max="30" step="1" pattern="%.1f °C" readOnly="false"/>
+        <state min="15" max="25" step="1" pattern="%.1f °C" readOnly="false"/>
     </channel-type>
     
-     <channel-type id="save_temp" advanced="true">
+     <channel-type id="save_temp_setpoint" advanced="true">
         <item-type>Number</item-type>
-        <label>Save</label>
+        <label>Save Mode Temperature Setpoint</label>
         <description>Target temperature of save mode</description>
         <category>Temperature</category>
-        <state  min="4.5" max="30" step="1" pattern="%.1f °C" readOnly="false"/>
+        <state  min="5" max="15" step="1" pattern="%.1f °C" readOnly="false"/>
     </channel-type>
     
-    <channel-type id="room_temp" advanced="true">
+    <channel-type id="room_temp_setpoint">
         <item-type>Number</item-type>
-        <label>Room</label>
+        <label>Room Temperature Setpoint</label>
         <description>Target temperature of rooms</description>
         <category>Temperature</category>
-        <state  min="4.5" max="30" step="1" pattern="%.1f °C" readOnly="false"/>
+        <state  min="15" max="25" step="1" pattern="%.1f °C" readOnly="false"/>

```

Comment:
Well, I personally felt that a room temperature target of 4.5°C minimum makes no sense. It´s far too cold.
Same for max 30°C: too hot